cpp/tests/test_segmentation_image.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include <filesystem>
+#include <gtest/gtest.h>
+
+#include <deplex/plane_extractor.h>
+#include <deplex/utils/depth_image.h>
+#include <deplex/utils/eigen_io.h>
+
+TEST(ReadImage, count_image) {
+  const uint iterations = 500;
+  const std::string IMAGE = "icl_nuim";
+
+  std::__fs::filesystem::path data_dir = "/Users/denisovlev/Documents/CLionProjects/deplex/dataBenchmark";
```

Comment:
Hardcoded path...

Reply:
Will be deleted later

Reply:
With this "local" test I can't see whether CI successfully passes all other tests. We'll have to delete it before moving further.